packages/rest-hooks/src/internal.ts

```diff
@@ -5,5 +5,5 @@ export {
   DispatchContext,
   hasUsableData,
 } from '@rest-hooks/core';
-export const buildInferredResults = __INTERNAL__.buildInferredResults;
```

Comment:
doesn’t this mean it’s a breaking change?

Reply:
__INTERNAL__ is sort of like 'unstable'. current policy (up for debate) is to make breaking change on those bump minor versions only.

Reply:
Keeping old export and just adding the new one